docs/components/progress.md

```diff
@@ -0,0 +1,37 @@
+# Progress
+
+Progress elements provide feedback for determinate-length tasks.
+
+## Visual Examples
+
+### Default
+
+Three different formats are available to suit space constraints and desired aesthetic.
+
+| Bar | Column | Ring | 
+| :---: | :---: | :---: |
+| ![](https://drive.google.com/uc?id=1pxy538EdnryJNY_nXPUQIu5ovEObBN6V) | ![](https://drive.google.com/uc?id=1SFwRX_QwrCRB4vaRS8DLmGVZLrzs2izz) | ![](https://drive.google.com/uc?id=1o_iJMIN6QZWrhlozNdFHNCKp3lHN500H) | 
+
+
+### With additional feedback
+
+| Bar with text | Column with text | Ring with tooltip | 
+| --- | --- | --- |
+| ![Bar with text](https://drive.google.com/uc?id=1VWaMGVwRe6eT8M3_AQSpSV6q3-eLwFI4) |  ![Column with text](https://drive.google.com/uc?id=1hJG88hJdbX4U3w5knBeflKyqLprQ-GTd) |     ![Ring with tooltip](https://drive.google.com/uc?id=1DuaD3DEtee1h2dSnZWXgn8Pvw588iYyF) |
```

Comment:
Can we make it clearer that these placements are just examples? I think it will depend a lot on the other elements and layout, so not sure we want to get very prescriptive on this part.

Reply:
Ah, good point. I made a few updates. 

I left this in the Vis Examples section for now, but I could imagine this moving into the "Best Practice" section in the future since the supporting info isn't intrinsic to the progress element.